### DISCUSSION TITLE
List of environment variables

### DISCUSSION THREAD

**Discussion by insuusvenerati** [itzg/docker-minecraft-server#1847] [Q&A] [answered]
Hi! I'm working on templating in docker compose so I can more easily spin up varying servers. Is there a list of all the env vars that are possible with this image? I tried digging through scripts, the dockerfile and readme but it's split up a lot. If you're able to easily make one or point me in the right direction I would be really grateful. Thanks for continuously maintaining this project. I've been using it for years.

**Comment by itzg** [comment, marked as answer]
Unfortunately not yet. I actually started on a tool that would pick out all of the variables and the defaults, when present...but only just a start 😀.

**Reply by insuusvenerati** [reply]
If the tool is open I'd be happy to try and help. Not a perfect go dev but I use it occasionally

**Reply by itzg** [reply]
It will be open sourced, but only barely started and sitting on my local drive somewhere 😁. It was started in Java, if that happens to be more approachable, but Go is good too for that kind of tooling.
